File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/VedtakA009.java

package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;
import lombok.Data;
import lombok.EqualsAndHashCode;

@EqualsAndHashCode(callSuper = true)
@JsonInclude(Include.NON_NULL)
@Data
public class VedtakA009 extends Vedtak{

    private String artikkelforordning;

    private String erendringsvedtak;

    private Periode gjelderperiode;

    private String gjeldervarighetyrkesaktivitet;

    private String land;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/VedtakA009.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class VedtakA009(
    var artikkelforordning: String? = null,
    var gjelderperiode: Periode? = null,
    var gjeldervarighetyrkesaktivitet: String? = null,
    var land: String? = null
) : Vedtak()
